plugins/dbus_announce.c

```diff
@@ -0,0 +1,144 @@
+#include "system.h"
+
+#include <dbus/dbus.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <rpm/rpmlog.h>
+#include <rpm/rpmts.h>
+#include <rpm/rpmdb.h>
+#include "lib/rpmplugin.h"
+
+struct dbus_announce_data {
+    DBusConnection * bus;
+    int old_dboffset;
```

Comment:
old_dboffset isn't used so should be removed

Reply:
Done.